Stronger/StrongerDelegate.swift

```diff
@@ -76,7 +76,15 @@ class StrongerDelegate: SpeziAppDelegate {
     private var healthKit: HealthKit {
         HealthKit {
             CollectSample(
-                HKQuantityType(.stepCount),
+                HKQuantityType(.appleExerciseTime),
+                deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
+            )
+            CollectSample(
+                HKQuantityType(.dietaryProtein),
+                deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
+            )
+            CollectSample(
+                HKWorkoutType.workoutType(),
```

Comment:
I think that this line is not a real HKWorkout Type, not sure what you wanted to add here?

Reply:
Hi! Since we wanted to collect data on resistance training, I wanted a way to tell what kind of workout a person is currently doing (cardio, resistance, etc)